### PR TITLE
[doc] Add doc for pre-sql and post-sql

### LINKED ISSUE #10233: [Feature][doc] Add an example about how to use pre_sql post_sql in SQL task
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/guide/task/sql.html 

we should add an example about how to use presql and postsql in sql task, some users will use some select sql in pre/post sql and cause error

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by stalary**
## Purpose of the pull request
add doc for pre-sql and post-sql
close https://github.com/apache/dolphinscheduler/issues/10233

### CODE REVIEW

**Review by ruanwenjun — DISMISSED** [review]
LGTM,

**Review by SbloodyS — DISMISSED** [review]
+1

File: docs/docs/en/guide/task/sql.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ Log in to the bigdata cluster and use 'hive' command or 'beeline' or 'JDBC' and
 
 ![hive-sql](../../../../img/tasks/demo/hive-result.png)
 
+### Use Pre-SQL and Post-SQL
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/sql.md]: It is seem this use case is not related to above, can we and the other section for above docs? I mean we should

```diff
+ ### Hive Table Create Example
+
- ### Create a Temporary Table in Hive and Write Data
+ #### Create a Temporary Table in Hive and Write Data
This example creates a temporary table `tmp_hello_world` in Hive and writes a row of data. Before creating a temporary table, we need to ensure that the table does not exist. So we use custom parameters to obtain the time of the day as the suffix of the table name every time we run, this task can run every different day. The format of the created table name is: `tmp_hello_world_{yyyyMMdd}`.
![hive-sql](../../../../img/tasks/demo/hive-sql.png)
- ### After Running the Task Successfully, Query the Results in Hive
+ #### After Running the Task Successfully, Query the Results in Hive
Log in to the bigdata cluster and use 'hive' command or 'beeline' or 'JDBC' and other methods to connect to the 'Apache Hive' for the query. The query SQL is `select * from tmp_hello_world_{yyyyMMdd}`, please replace `{yyyyMMdd}` with the date of the running day. The following shows the query screenshot:

 ![hive-sql](../../../../img/tasks/demo/hive-result.png)
```

Comment (stalary) [reply docs/docs/en/guide/task/sql.md]: Thank you, I will modify it

Comment (stalary) [reply docs/docs/en/guide/task/sql.md]: done.

**Review by zhongjiajie — APPROVED** [review]
LGTM thanks @stalary
